src/biophysics.jl

```diff
@@ -398,14 +468,15 @@ Note that there is no recovery of heat or moisture assumed in the nose.
     - `fN2`; fractional N2 concentration in atmosphere, -
     ...    
 """
-function resp(
+function respiration(
     T_x = 296.15K,
     Q_metab = 0.01241022W,
     fO2_ext = 0.20,
     pant = 1,
     rq = 0.8,
     T_air = 293.15K,
     rh = 50,
+    elev = 0m,
```

Comment:
good catch on this but I think we should put it as 'nothing' here and we can add a note in the help to say that if you know the atmospheric pressure, elev won't be used, but if you don't know atmospheric pressure, elev will be used to estimate it

Reply:
fixed.